recipe/meta.yaml

```
@@ -1,32 +1,38 @@
-{% set version = "4.1" %}
+{% set version = "5.1" %}
 
 package:
   name: django
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/D/Django/Django-{{ version }}.tar.gz
-  sha256: 032f8a6fc7cf05ccd1214e4a2e21dfcd6a23b9d575c6573cacc8c67828dbe642
-
+  sha256: 848a5980e8efb76eea70872fb0e4bc5e371619c70fffbe48e3e1b50b2c09455d
+# python-tzdata is a fallback package for systems not providing timezone data, but the conda python package depends on tzdata.
+# Therefore we do not need to depend on python-tzdata.
+  patches:
+    - patches/001-Win-tzdata-removal.patch   # [win]
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
-  skip: True # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
+  skip: True  # [py<310]
   entry_points:
     - django-admin = django.core.management:execute_from_command_line
 
 requirements:
+  build:
+    - patch   # [unix]
+    - m2-patch  # [win]
   host:
     - python
     - pip
-    - setuptools >=40.8.0
+    - setuptools
     - wheel
   run:
-    - asgiref >=3.5.2,<4
-    - backports.zoneinfo # [py<39]
     - python
-    - sqlparse >=0.2.2
-    - python-tzdata # [win]
+    - asgiref >=3.8.1
+    - sqlparse >=0.3.1
```

```suggestion
    - sqlparse >=0.3.1
  run_constrained:
    - argon2-cffi >=19.1.0
```
See https://github.com/django/django/blob/5.1/pyproject.toml#L41C11-L41C30